src/Core/src/Handlers/Editor/EditorHandler.Android.cs

```diff
@@ -102,8 +102,12 @@ public static void MapPlaceholderColor(IEditorHandler handler, IEditor editor)
 		public static void MapVerticalTextAlignment(IEditorHandler handler, IEditor editor) =>
 			handler.PlatformView?.UpdateVerticalTextAlignment(editor);
 
-		public static void MapKeyboard(IEditorHandler handler, IEditor editor) =>
+		public static void MapKeyboard(IEditorHandler handler, IEditor editor)
+		{
+			handler.UpdateValue(nameof(IEntry.Text));
```

Comment:
Did you mean IEditor here?
```suggestion
			handler.UpdateValue(nameof(IEditor.Text));
```

Reply:
I did :) Thanks